tasks/Task.cpp

```diff
@@ -263,7 +263,7 @@ static gps_base::SatelliteInfo convertToBaseSatelliteInfo(const SatelliteInfo &s
         gps_base::Satellite sat;
         sat.azimuth = (int)data.azimuth.getDeg();
         sat.elevation = (int)data.elevation.getDeg();
-        sat.PRN = data.satellite_id;
+        sat.PRN = (static_cast<uint32_t>(data.gnss_id) << 8) + data.satellite_id;
```

Comment:
- what means PRN (SNR) ?

Reply:
The PRN is a code that is different for each GPS satellite. It's used to encode the signal of each GPS satellite differently, allowing to separate them. Because of its unicity, it can also be used to identify the same satellite in different constellation reports. Note that historically, a satellite has already changed its PRN (but rarely, so the tracking still works over a few days, which is truly our purpose here).

In the software, this is a misuse of the name - but changing the name of the field in the structure would cost too much. It's essentially used as a satellite ID, allowing to track how the overall GNSS constellation is changing.

Reply:
nice. thanks for the explanation!